net-libs/libblkmaker/libblkmaker-0.5.3.recipe

```diff
@@ -0,0 +1,65 @@
+SUMMARY="A library implementing getblocktemplate"
+DESCRIPTION="libblkmaker provides an implementation of getblocktemplate, the\
+ current Bitcoin mining protocol."
+HOMEPAGE="https://github.com/bitcoin/libblkmaker"
+COPYRIGHT="Luke Dashjr"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/bitcoin/libblkmaker/archive/v${portVersion}.tar.gz"
```

Comment:
You might wish to drop the curly braces in `v${portVersion}.tar.gz` as they are not required here.